doc/vxlan/EVPN/EVPN_VXLAN_HLD.md

```diff
@@ -977,12 +978,12 @@ Kernel uses a single flag for both ARP & ND suppression, We support only one CLI
 
 With the L2 and L3 support over VXLAN it is useful to support tunnel statistics. 
 
-#### 4.3.11.1 Counters supported. 
+#### 4.3.11.1 Counters supported.
 
 The Packets Rx/Tx and Octets Rx/Tx will be counted per tunnel.
 These will be stored in the counters DB for each tunnel. 
 
-####  4.3.11.2 Changes to SwSS
+#### 4.3.11.2 Changes to SwSS
```

Comment:
1. What would happen if only a V4 VTEP is configured and an EVPN route with the NEXT_HOP as v6 address is received ? 
2. For co-existence of V4 and V6 VTEPs,
   2a.  EVPN routes be advertised with both V4 and V6 addresses as  next hops.
   2b. On reception of 2 different routes from the same VTEP a receiving VTEP should prefer only the V6 advertisement if the receiver VTEP is capable and configured for both V4 and V6 addresses.

Reply:
1: Those routes should still be installed. In one direction traffic would be encapsulated with an IPv6 underlay address, the other direction IPv4
2a: V4/V6 VTEP in the same VTEP device would advertise both (not considered here)
2b: For the future, v6 should be preferred